walkthroughs/week-2-server/examples/random-quotes/src/main/webapp/script.js

```diff
@@ -56,6 +56,7 @@ function addQuoteToDom(quote) {
  * whichever syntax makes the most sense to you.
  */
 function getRandomQuoteUsingArrowFunctions() {
+  console.log("Hey mate")
```

Comment:
Nit: We should remove this log statement or change it to be more descriptive.

Reply:
Okay, I removed it!